susemanager-utils/testing/automation/push-to-obs.sh

```diff
@@ -73,6 +72,8 @@ echo "Starting building and submission at $(date)"
 date
 [ -d ${GITROOT}/logs ] || mkdir ${GITROOT}/logs
 for p in ${PACKAGES};do
+    pkg_dir=$(cat rel-eng/packages/${p} | tr -s " " | cut -d" " -f 2)
+    CHOWN_CMD="chown -f -R $(id -u):$(id -g) /manager/$pkg_dir"
```

Comment:
```suggestion
    CHOWN_CMD_FULL="${CHOWN_CMD}; chown -f -R $(id -u):$(id -g) /manager/$pkg_dir"
```
And then of course change the `docker_run` call below.